test/utils.sh

```diff
@@ -170,25 +174,25 @@ waitBuildExpectedStatus() {
 export -f waitBuildExpectedStatus
 
 
-## Common env parameters
-export FISSION_NAMESPACE=${FISSION_NAMESPACE:-fission}
-export FUNCTION_NAMESPACE=${FUNCTION_NAMESPACE:-fission-function}
+# ## Common env parameters
+# export FISSION_NAMESPACE=${FISSION_NAMESPACE:-fission}
```

Comment:
Will we be using this later for other tests?

Reply:
No, it's not required, so removed.